test/thermo/CoverageDependentSurfPhase_Test.cpp

```diff
@@ -0,0 +1,512 @@
+#include "gtest/gtest.h"
+#include "cantera/thermo/CoverageDependentSurfPhase.h"
+#include "cantera/thermo/ThermoFactory.h"
+
+
+namespace Cantera
+{
+
+class CoverageDependentSurfPhase_Test: public testing::Test
+{
+public:
+    CoverageDependentSurfPhase_Test() {
+        test_phase.reset(newPhase("copt_covdepsurf_example.yaml", "coverages_0"));
+    }
+    std::unique_ptr<ThermoPhase> test_phase;
+    CoverageDependentSurfPhase* covdepsurf;
+};
+
+TEST_F(CoverageDependentSurfPhase_Test, construct_from_file)
+{
+    CoverageDependentSurfPhase* coveragedependentsurf_phase = dynamic_cast<CoverageDependentSurfPhase*>(test_phase.get());
+    EXPECT_TRUE(coveragedependentsurf_phase != NULL);
+}
+
+TEST_F(CoverageDependentSurfPhase_Test, reference_enthalpies_RT)
+{
+    const double expected_result[10] = {
+        4.895991737500073e-05,
+        4.617646334628971e-05,
+        5.963833377856196e-05,
+        5.692092800682202e-05,
+        -9.815993386192746e-05,
+        6.644423752958034e-05,
+        5.2492790438840376e-05,
+        6.225436283744746e-05,
+        -0.0035000916056617164,
+        5.94838246127799e-05
+    };
```

Comment:
All all of these values just regression tests? How do you know these values are correct?